userdocs/profiles.dev/docs/author-docs/kustomize-and-raw.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 4
+---
+
+# Kustomize patches and raw yaml artifacts
+
+Both Kustomize patches and raw yamls (such as a simple deployment manifest,
+or any other Kubernetes object) can be added to a profile under the same key.
+
+The resources must be stored locally within the profile directory.
+For example:
+
+```bash
+org-profiles-repo/
+├── our-awesome-apps-profile
+│   ├── super-cool-artifact-manifests
+│   │   ├── deployment.yaml
+│   │   └── patches.yaml
+│   └── profile.yaml
+...
+```
+
+Then in the `profile.yaml` we add these artifact manifests by using the `kustomize`
+type identifier:
+
+```yaml
+# ...
+spec:
+  # ...
+  artifacts:
+    - name: # the name of your artifact as you would like it to be known in the profile
+      kustomize:
+        path: "super-cool-artifact-manifests/*" # the relative path to the manifests directory
+    # ...
+```
+
+The exact directory structure can be as you wish, so log as it is child to the profile
```

Comment:
```suggestion
The exact directory structure can be as you wish, so long as it is a child to the profile
```

Reply:
i omitted the `a` on purpose as it is valid english, but i can put it back in for you 😜

Reply:
haha, it's fine, if you don't want to :D